GOVERNANCE.md

```diff
@@ -50,59 +46,140 @@ and so on.
 * Any committer can decide to close a PR or issue if they determine the change doesn't suit the project.
 * Appium has a large ecosystem of repositories. Some are 'core' in the sense of being central subpackages with a lot of traffic like `appium/appium` or `appium/appium-ios-driver`. Others are 'peripheral' in the sense of receiving few changes, being maintained by 1 person or no one, etc..., like `appium/ruby_lib` or `appium/appium_thor` For 'core' repos, committers should not merge their own code straightaway. Like all contributors they should open a PR and get a +1 from another committer. For 'peripheral' repos for which they are the sole maintainer or for which they have a good understanding, it is obtuse to insist on another contributor's +1 and this rule should be relaxed in that case. In sum: use good judgment and don't ram code through without a review when you can help it.
 
-The current committers are (in addition to the TC members above):
+The current committers are (along with their primary areas(s) of ownership or maintenance):
 
-* [@hugs](https://github.com/hugs)
-* [@filmaj](https://github.com/filmaj)
-* [@aluedeke](https://github.com/aluedeke)
-* [@sebv](https://github.com/sebv)
-* [@vgrigoruk](https://github.com/vgrigoruk)
-* [@DylanLacey](https://github.com/DylanLacey)
-* [@maudineormsby](https://github.com/maudineormsby)
-* [@titusfortner](https://github.com/titusfortner)
-* [@bayandin](https://github.com/bayandin)
-* [@jdeff](https://github.com/jdeff)
-* [@moizjv](https://github.com/moizjv)
-* [@gempesaw](https://github.com/gempesaw)
-* [@Jonahss](https://github.com/Jonahss)
-* [@JamieEdge](https://github.com/JamieEdge)
-* [@saikrishna321](https://github.com/saikrishna321)
-* [@sravanmedarapu](https://github.com/sravanmedarapu)
-* [@peepa](https://github.com/peepa)
-* [@scottdixon](https://github.com/scottdixon)
-* [@drpy](https://github.com/drpy)
-* [@Astro03](https://github.com/Astro03)
-* [@sbonebrake](https://github.com/sbonebrake)
-* [@SrinivasanTarget](https://github.com/SrinivasanTarget)
-* [@triager](https://github.com/triager)
-* [@ddkjin](https://github.com/ddkjin)
-* [@stuartbrussell-intuit](https://github.com/stuartbrussell-intuit)
-* [@vmaxim](https://github.com/vmaxim)
-* [@Dan-Maor](https://github.com/Dan-Maor)
-* [@laolubenson](https://github.com/laolubenson)
-* [@Dor-bl](https://github.com/Dor-bl)
-* [@boneskull](https://github.com/boneskull)
+* [@dan-maor](https://github.com/Dan-Maor) - XCUITest driver and iOS internals
+* [@dor-bl](https://github.com/Dor-bl) - DotNet client
+* [@eglitise](https://github.com/eglitise) - Appium server, Inspector
+* [@jlipps](https://github.com/jlipps) - Appium server, Inspector
+* [@kazucocoa](https://github.com/KazuCocoa) - all Appium projects
+* [@laolubenson](https://github.com/laolubenson) - DotNet client
 * [@mwakizaka](https://github.com/mwakizaka)
```

Comment:
Let me modify the list later as:

mwakizaka and rerorero are xcuitest driver and uia2/espresso driver (JS code ones) related.
tomriddly is appium-ios-driver related.


And noticed we could add these persons as well. I'll create a new PR later after this merge for them:
https://github.com/appium/java-client/commits?author=valfirst
https://github.com/appium/appium-espresso-driver/commits?author=pr4bh4sh

Reply:
great, thank you!